python/desc/imsim/imSim.py

```diff
@@ -693,3 +675,28 @@ def add_cosmic_rays(gs_interpreter, phot_params):
             galsim.Image(crs.paint(imarr, exptime=exptime), wcs=image.wcs)
 
     return None
+
+
+def add_treering_info(gs_interpreter):
+    """
+    Adds tree ring info based on a model derived from measured sensors.
+
+    Parameters
+    ----------
+    gs_interpreter: lsst.sims.GalSimInterface.GalSimInterpreter
+        The object that is actually drawing the images
+
+    Returns
+    -------
+    None
+        Will act on gs_interpreter, adding tree ring information to the detectors.
+    """
+    tr_filename = os.path.join(lsstUtils.getPackageDir('imsim'),
+                               'data', 'tree_ring_data', 'tree_ring_parameters_2018-04-24.txt')
```

Comment:
Should we hardcode this tree ring file?  Maybe it should be in the config file?

Reply:
I thought about that, but I didn't bother since I figured that we would change this very infrequently.  If we decide we need that sort of runtime flexibility, we can put it into the config then.